apps/dashboard/app/helpers/application_helper.rb

```diff
@@ -48,9 +50,9 @@ def fa_icon(icon, fa_style: "fas", id: "")
   end
 
   def app_icon_tag(app)
-    if app.icon_path.file?
+    if app.png_icon? || app.svg_icon?
```

Comment:
This file seems to have a lot of updates but this is the only meaningful one. Can you clean this up a bit?